classes/Translate.php

```diff
@@ -232,7 +233,9 @@ public static function getModuleTranslation(
                 $defaultKeyFile = strtolower('<{' . $name . '}prestashop>' . $file) . '_' . $key;
             }
 
-            if (isset($currentKeyFile) && !empty($_MODULES[$currentKeyFile])) {
+            if($translationsMerged[$name][$iso] === false){
+                $ret = stripslashes($string);
```

Comment:
Why not using the else statement?

Reply:
I am not sure how you would do this with "else" at this point and still achieve the same. 
I added the "if" at this point because _MODULES is not empty but will contain wrong information in my testcase.